tests/by-util/test_shuf.rs

```diff
@@ -1 +1,55 @@
-// ToDO: add tests
+// ToDO: add more tests
```

Comment:
kudos for the tests :)

it would be nice to also test success and not only failures ;)

Reply:
I think @jaggededgedjustice said that they would like to do it in #1859, and I didn't want to step on any toes 😄

Reply:
That's correct, tests will be along soon.